unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWNftsShould.cs

```diff
@@ -79,7 +79,7 @@ public void NftComponent()
     protected override IEnumerator TearDown()
     {
         BIWCatalogManager.ClearCatalog();
-        BuilderInWorldNFTController.i.ClearNFTs();
+        BIWNFTController.i.ClearNFTs();
```

Comment:
```suggestion
        BIWNFTController.i.ClearNFTs();
```
This singleton makes me cringe everytime I see it 😢